### ISSUE TITLE
npm

### ISSUE THREAD

**Issue by jamesgpearce** [robrighter/node-xml#4]
Shouldn't this be in npm? ;-)

**Comment by jupiter** [comment]
Yes, would be great if you can publish on npm repository.

**Comment by hdorio** [comment]
yeah maybe this is a good start ? https://github.com/hammerdr/node-xml/commit/856ce992c7058424e9475bc54d414ea0a59e80ab

**Comment by edubkendo** [comment]
Just wondering why this still isn't an npm package?

**Comment by robrighter** [comment]
It's been in npm for a while now:

https://npmjs.org/package/node-xml

On Sep 14, 2012, at 7:33 PM, edubkendo notifications@github.com wrote:

Just wondering why this still isn't an npm package?

—
Reply to this email directly or view it on
GitHubhttps://github.com/robrighter/node-xml/issues/4#issuecomment-8578928.
